node/src/accountant/payment_adjuster/inner.rs

```diff
@@ -1,204 +1,277 @@
+        GutsOfPaymentAdjusterInner, PaymentAdjusterInner,
     };
-    use std::time::SystemTime;
+    use std::panic::{catch_unwind, AssertUnwindSafe};
 
     #[test]
-    fn inner_real_is_constructed_correctly() {
-        let now = SystemTime::now();
-        let transaction_fee_count_limit_opt = Some(3);
+    fn defaulted_payment_adjuster_inner() {
+        let subject = PaymentAdjusterInner::default();
+
+        let guts_is_none = subject.initialized_guts_opt.borrow().is_none();
+        assert_eq!(guts_is_none, true)
+    }
+
+    #[test]
+    fn initialization_and_getters_of_payment_adjuster_inner_work() {
+        let subject = PaymentAdjusterInner::default();
+        let tx_count_limit_opt = Some(3);
         let cw_service_fee_balance = 123_456_789;
-        let largest_exceeding_balance_recently_qualified = 44_555_666;
-        let result = PaymentAdjusterInnerReal::new(
-            now,
-            transaction_fee_count_limit_opt,
+        let max_debt_above_threshold_in_qualified_payables_minor = 44_555_666;
+
+        subject.initialize_guts(
+            tx_count_limit_opt,
             cw_service_fee_balance,
-            largest_exceeding_balance_recently_qualified,
+            max_debt_above_threshold_in_qualified_payables_minor,
         );
+        let read_max_debt_above_threshold_in_qualified_payables_minor =
+            subject.max_debt_above_threshold_in_qualified_payables_minor();
+        let read_tx_count_limit_opt = subject.transaction_count_limit_opt();
+        let read_original_cw_service_fee_balance_minor =
+            subject.original_cw_service_fee_balance_minor();
+        let read_remaining_cw_service_fee_balance_minor =
+            subject.remaining_cw_service_fee_balance_minor();
 
-        assert_eq!(result.now, now);
         assert_eq!(
-            result.transaction_fee_count_limit_opt,
-            transaction_fee_count_limit_opt
+            read_max_debt_above_threshold_in_qualified_payables_minor,
+            max_debt_above_threshold_in_qualified_payables_minor
         );
+        assert_eq!(read_tx_count_limit_opt, tx_count_limit_opt);
         assert_eq!(
-            result.original_cw_service_fee_balance_minor,
+            read_original_cw_service_fee_balance_minor,
             cw_service_fee_balance
         );
         assert_eq!(
-            result.unallocated_cw_service_fee_balance_minor,
+            read_remaining_cw_service_fee_balance_minor,
             cw_service_fee_balance
         );
-        assert_eq!(
-            result.largest_exceeding_balance_recently_qualified,
-            largest_exceeding_balance_recently_qualified
-        )
     }
 
     #[test]
-    #[should_panic(
-        expected = "Broken code: Called the null implementation of the now() method in PaymentAdjusterInner"
-    )]
-    fn inner_null_calling_now() {
-        let subject = PaymentAdjusterInnerNull::default();
+    fn reducing_remaining_cw_service_fee_balance_works() {
```

Comment:
Just a small change. I think you can use `subtracting` instead of `reducing`. It'll rhyme with the act of the test.